nearcore/src/entity_debug_serializer.rs

```diff
@@ -0,0 +1,487 @@
+/// This file implements a completely new serialization format which
+/// flattens anything that implements serde::Serialize into an
```

Comment:
> flattens

How does it handle structs with fields of the same type?
For example, a struct containing two BlockInfos?